rpcs3/Emu/Cell/Modules/cellRtc.cpp

```diff
@@ -761,72 +761,47 @@ error_code cellRtcGetTick(vm::cptr<CellRtcDateTime> pTime, vm::ptr<CellRtcTick>
 
 	if (!pTick)
 	{
-		return CELL_RTC_ERROR_INVALID_POINTER;
-	}
-
-	s64 days_in_years = ((((pTime->year * 365ULL) + ((pTime->year + 3) / 4)) - ((pTime->year + 99) / 100)) + ((pTime->year + 399) / 400) + -366);
-
-	// 1-12
-	if (1 < pTime->month)
-	{
-		u32 month_idx         = pTime->month - 1;
-		u32 monthIdx_adjusted = is_leap_year(pTime->year) * 12;
-		do
-		{
-			days_in_years += DAYS_IN_MONTH[monthIdx_adjusted];
-			month_idx -= 1;
-			monthIdx_adjusted += 1;
-		} while (month_idx != 0);
+		return CELL_OK;
```

Comment:
This is unreachable code as vm::check_addr fails before for nullptr.

Reply:
In the real function, it checks syscall_sys_memory_get_page_attribute (which i assume also checks nullptr) and is placed where our vm::check_addr is placed.
The nullptr check is just a sanity check in the original function I think.
It might as well be removed here.